src/lib/i18n/translations/locales/en.ts

```diff
@@ -0,0 +1,254 @@
+import k, { TranslationDef } from "../keys"
+
+const def: TranslationDef = {
```

Comment:
I'd recommend defining translations in JSON files for better interop with other tools.